Solutions/Corvus.Retry.Specs/Corvus.Retry.Specs.csproj

```diff
@@ -4,12 +4,23 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <LangVersion>latest</LangVersion>
     <IsPackable>false</IsPackable>
+    <Copyright>Copyright (c) Endjin Limited 2020. All rights reserved.</Copyright>
```

Comment:
Not harmful not not strictly required either because `Endjin.RecommendedPractices` sets this if you don't.